### PR TITLE
SPCTR-3167: Fix - How To block heading responsive font-size issue

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Heading font-size is not working on mobile and tablet as intended.
https://brainstormforce.atlassian.net/browse/SPCTR-3167
### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -216,6 +216,7 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 ## Changelog ##
 
 ### x.x.x ###
+* Fix: How To - Resolved an issue in which heading responsive font-size would not work as intended.
```
Comment (bsf-zanev) [inline README.md:219]:
```suggestion
* Fix: How To - Resolved an issue in which the responsive heading font-sizes would not work as intended. 
```

File: src/blocks/how-to/styling.js [hunk 2]
```diff
@@ -385,6 +385,12 @@ function styling( attributes, clientId, name, deviceType ) {
 		},
 	};
 
+	mobileSelectors[ ' ' + headingTag + '.block-editor-rich-text__editable.uagb-howto-heading-text' ] = {
```
Comment (bsf-zanev) [inline src/blocks/how-to/styling.js:388]:
```suggestion
	mobileSelectors[ ` ${ headingTag }.block-editor-rich-text__editable.uagb-howto-heading-text` ] = {
```

File: src/blocks/how-to/styling.js [hunk 3]
```diff
@@ -470,6 +476,12 @@ function styling( attributes, clientId, name, deviceType ) {
 		},
 	};
 
+	tabletSelectors[ ' ' + headingTag + '.block-editor-rich-text__editable.uagb-howto-heading-text' ] = {
```
Comment (bsf-zanev) [inline src/blocks/how-to/styling.js:479]:
```suggestion
	tabletSelectors[ ` ${ headingTag }.block-editor-rich-text__editable.uagb-howto-heading-text` ] = {
```

File: README.md (hunk 1 above)
Comment (rajmohanreddy774) [reply README.md]: Done.

File: src/blocks/how-to/styling.js (hunk 2 above)
Comment (rajmohanreddy774) [reply src/blocks/how-to/styling.js]: Updated.

File: src/blocks/how-to/styling.js (hunk 3 above)
Comment (rajmohanreddy774) [reply src/blocks/how-to/styling.js]: updated.

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

**Pull request merged**